.github/dependabot.yml

```diff
@@ -7,11 +7,17 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "monthly"
-    groups:
-      aws-sdk:
-        patterns:
-          - '@aws-sdk/*'
+
+    # We've seen issues with versions of `@aws-sdk/client-s3` later than 3.703.0 where put object requests fail with:
+    #  > RequestTimeout: Your socket connection to the server was not read from or written to within the timeout period. Idle connections will be closed.
+    # Attempts to increase the timeout to 10 minutes was not successful for all cases.
```

Comment:
> Attempts to increase the timeout to 10 minutes

I thought it was 10 **seconds**?

Reply:
Oops! Yes, I cannot read my own comment... Will update.